File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer: set = set()
    numbers.sort()
    for i in range(len(numbers)-1):
        for j in range(i, len(numbers)):
            if i == j:
                continue
            answer.add(numbers[i] + numbers[j])
    return sorted(list(answer))

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    var result = new Set();
    numbers.map(function (num, index) {
        numbers.map(function (num2, index2) {
            if (index !== index2) {
                result.add(num + num2);
            }
        });
    });
    return Array.from(result).sort(function (a, b) { return a - b; });
}